src/functTest/groovy/com/bmuschko/gradle/docker/DockerWorkflowFunctionalTest.groovy

```diff
@@ -314,8 +317,8 @@ class DockerWorkflowFunctionalTest extends AbstractFunctionalTest {
     }
 
     def "Can build an image only once"() {
-        File imageDir = temporaryFolder.newFolder('images', 'minimal')
-        createDockerfile(imageDir)
+        File imageDir = new File(projectDir, 'images/minimal')
```

Comment:
As soon as a test would fail the temporary folder would get blown away and all downstream tests, or those tests which have yet to run, would fail because the folder no longer exists. Reading up on TemporaryFolder this is expected behavior.